src/Compilers/Test/Core/CompilationVerifier.cs

```diff
@@ -300,20 +300,30 @@ public void VerifyTypeIL(string typeName, Action<string> validateExpected)
             }
         }
 
-        private sealed class Resolver : ILVerify.ResolverBase
+        private sealed class Resolver : ILVerify.IResolver
```

Comment:
:memo: The changes in this file are to adapt to new ILVerify interface. 
The change from `Immutable<byte>` to `PEReader` in the dictionary below is to avoid a caching issue whereby ILVerify ended up with two different PEReaders for the corlib (causing many failures).

Reply:
Can you elaborate on this? I can't see from the code how their could be a caching issue or how this fixes it. The `PEReader` is just a thin wrapper over the array.

Reply:
We used to make `PEReader` instances on demand, but ILVerify was calling us twice for `netstandard` (because of the `SetSystemModuleName` call, if I recall correctly) and was ending up with two `PEReader` instances for the same module. Now, we make sure to return the same `PEReader` for both callbacks from ILVerify for `netstandard` by caching the `PEReader` instances directly.